### PR TITLE
Update owners file

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,10 +1,13 @@
 approvers:
 - bparees
+- awgreene
+- perdasilva
 - joelanford
 - perdasilva
```
Comment (bparees) [inline OWNERS:6]: i don't think we need Per twice :)

maybe alpha-sort the list to help avoid that problem in the future.

File: OWNERS [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-- timflannagan
 reviewers:
-- exdx
-- tylerslaton
-- timflannagan
+- dtfranz
+- awgreene
+- perdasilva
+- ankitathomas
+- tmshort
```
Comment (bparees) [inline OWNERS:12]: alpha sort

File: OWNERS (hunk 1 above)
Comment (bparees) [reply OWNERS:6]: (though if we had two of Per, that would certainly be awesome)

Comment (awgreene) [reply OWNERS:6]: Ha! Yes it would!

**Pull request merged**